spec/immutable-data-structures.html

```diff
@@ -275,7 +275,7 @@ <h1>Tuple.prototype.sliced ( _start_, _end_ )</h1>
         <p>The following steps are taken:</p>
         <emu-alg>
           1. Let _T_ be ? thisTupleValue(*this* value).
-          1. Let _list_ be ? CreateListFromTuple(_T_)
+          1. Let _list_ be a new List containing the elements of _T_'s [[Sequence]] value.

```

Comment:
I guess in this case, you don't need a copy, and can just use T's [[Sequence]] value directly.

Reply:
that's true, I was liberal about creating new copies of a Tuple's [[Sequence]] but I don't really need to.